webrtc-stats.html

```diff
@@ -307,15 +277,46 @@ <h3>
       </section>
       <section>
         <h3>
-          Stats objects for deleted monitored objects
+          Lifetime considerations for monitored objects
         </h3>
         <p>
-          When a monitored object is destroyed, a final stats object is produced, carrying the
-          values current at the time of destruction. This object will be returned, with a timestamp
-          reflecting the time of destruction, whenever getStats() is called, as long as the
-          PeerConnection exists. This is important in order to report consistently on short-lived
-          objects and to be able to consistently report totals over the lifetime of a
-          PeerConnection.
+          The object descriptions will say what the lifetime of a <a>monitored object</a> from the
+          perspective of stats is. When a monitored object is "deleted", it no longer appears in stats;
+          until this happens, it will appear. This may or may not correspond to the actual lifetime
+          of an object in an implementation; what matters for this specification is what appears in stats.
+        </p>
+        <p>
+          If a monitored object can only exist
+          in a few instances over the lifetime of a PeerConnection, it may be simplest to consider it
+          "eternal" and never delete it from the set of objects reported on in stats.
+          This type of object will remain visible until the PeerConnection
+          is no longer available; it is also visible in getStats() after pc.close(). This is the default
+          when no lifetime is mentioned in its specification.
+        </p>
+        <p>
+          Objects that might exist in many instances over time SHOULD have a defined time at which
+          they are deleted, at which time a "statsended" event is fired on their behalf. Each
+          event that causes deletions to happen SHOULD fire only one "statsended" event; for instance,
```

Comment:
link to "statsended" to the appropriate section in webrtc-pc

Reply:
Done.